library/std/src/os/unix/net/datagram.rs

```diff
@@ -854,8 +854,18 @@ impl UnixDatagram {
     ///
     /// # Examples
     ///
-    #[cfg_attr(any(target_os = "android", target_os = "linux"), doc = "```no_run")]
-    #[cfg_attr(not(any(target_os = "android", target_os = "linux")), doc = "```ignore")]
+    #[cfg_attr(
+        any(target_os = "android", target_os = "linux", all(target_os = "freebsd", freebsd13),),
+        doc = "```no_run"
+    )]
+    #[cfg_attr(
+        not(any(
+            target_os = "android",
+            target_os = "linux",
+            all(target_os = "freebsd", freebsd13)
```

Comment:
```suggestion
            target_os = "netbsd",
            all(target_os = "freebsd", freebsd13)
```

Reply:
```suggestion
            target_os = "freebsd"
```
And do the same on lines 884 and 896 as well.